releases: add 3.0.13 release notes (#2376)
pingcap/docs: releases/3.0.13.md

---
title: TiDB 3.0.13 Release Notes
category: Releases
---

# TiDB 3.0.13 Release Notes

Release date: April 22, 2020

TiDB version: 3.0.13

TiUP version: 3.0.13

## Bug Fixes

+ TiDB

>     - Fix the issue that the `INSERT ... ON DUPLICATE KEY UPDATE` statement might be executed incorrectly within a transaction because `MemBuffer` is not checked, when users need to insert multiple rows of duplicate data [#16690](https://github.com/pingcap/tidb/pull/16690)

Reviewer: ```suggestion
    - Fix the issue caused by unchecked `MemBuffer` that the `INSERT ... ON DUPLICATE KEY UPDATE` statement might be executed incorrectly within a transaction when users need to insert multiple rows of duplicate data [#16690](https://github.com/pingcap/tidb/pull/16690)
```
